coordinator/migrations/2023-07-11-104904_create_channels/up.sql

```diff
@@ -0,0 +1,28 @@
+-- Your SQL goes here
+CREATE TYPE "ChannelState_Type" AS ENUM (
+    'Pending',
+    'Open',
+    'Closed',
+    'ForceClosedRemote',
+    'ForceClosedLocal'
+);
+-- All transactions broadcasted by us
+CREATE TABLE "transactions" (
+    txid TEXT PRIMARY KEY,
+    -- the fee is stored here for simplicity of creating a sql query. However, it is not the source of truth and can
+    -- be recreated from looking up the transaction on the blockchain.
+    fee BIGINT NOT NULL DEFAULT 0,
```

Comment:
From the comment on line 9, the intention is to store all transactions published by _us_ anyway.

Reply:
hm... I wonder if that intention is feasible. 

My thinking: 
If we succeed in publishing a transaction we safe it ✅
But if we fail to publish it it depends *why* - if we fail because it is already published we should still safe it, but in a way that depicts that it was not published by us. 

(Note: In ItchySats we did not record it as failure when we noticed that a transaction that we tried to published was already published and hence the publication failed - I would treat this similar)

Reply:
It's a bit different in Lightning because the other party doesn't have the same commitment transaction. They have their "local" commitment transaction which is similar, but not exactly the same.

If you end up not being able to publish your local commitment transaction it's because the remote commitment transaction makes it on-chain first. In that case it would be incorrect to save your local commitment transaction.

Eventually we could save the remote commitment transaction as found on-chain, but that's probably outside of the scope of this PR which is already so big.